example/src/main/java/com/crowdin/platform/example/SettingsFragment.kt

```diff
@@ -44,12 +44,14 @@ class SettingsFragment : Fragment(), OnItemSelectedListener.SpinnerItemListener
 
         labels.add(languagePreferences.getLanguageCode())
 
-        manifestData?.languages?.forEach { languageIndex ->
-            Crowdin.getSupportedLanguages()?.data?.find { supportedLanguage ->
-                supportedLanguage.data.id == languageIndex
-            }?.let { languageInfoData ->
-                if (!labels.contains(languageInfoData.data.locale)) {
-                    labels.add(languageInfoData.data.locale)
+        manifestData?.languages?.let { languages ->
```

Comment:
Added lint config to ignore this lint warning. `example/lint.xml`. 
https://github.com/crowdin/mobile-sdk-android/pull/154/files

Reply:
fixed